asset/api/Asset.Samples.Tests/AssetFixture.cs

```diff
@@ -55,8 +52,16 @@ public AssetFixture()
 
     public void Dispose()
     {
-        _bucketFixture.Dispose();
-        _bigQueryClient.DeleteDataset(datasetId: DatasetId);
+        try
+        {
+            _bucketFixture.Dispose();
+            _bigQueryClient.DeleteDataset(
+              datasetId: DatasetId, options: new DeleteDatasetOptions() { DeleteContents = true });
```

Comment:
nit: Indents are always 4 spaces. Check here and everywhere.
```suggestion
            _bigQueryClient.DeleteDataset(
                datasetId: DatasetId, options: new DeleteDatasetOptions() { DeleteContents = true });
```